images/macos/scripts/build/install-ant.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash -e -o pipefail
+################################################################################
+##  File:  install-ant.sh
+##  Desc:  Install Apache Ant
+################################################################################
+
+source ~/utils/utils.sh
+
+if is_Monterey; then
```

Comment:
We don't really need here `if` block since execution is controlled by Packer template.